ReverseTicTacToe/UI/ConsoleUI.cs

```diff
@@ -33,32 +31,216 @@ public  void Init()
         }
         public  void Run()
         {
+            
             WriteLine("Welcome to Reverse Tic Tac Toe!" + Environment.NewLine);
-            getBoardSizeFromUser();
-            getGameTypeFromUser();
-            m_Game = new Game(m_BoardSize, m_GameType);
-            while(true)
+            int BoardSize =  getBoardSizeFromUser();
+            Game.eGameTypes GameType = getGameTypeFromUser();
+            m_Game = new Game(BoardSize, GameType);
+           // while(true)
+            //{
+            printGameBoard();
+            playTurn();
+            ReadLine();
+            
+            //}
+        }
+
+        private void playTurn()
+        {
+            int row = 0;
+            int col = 0;
+            //int return status: continue, place_taken, finished game
+            
+            if(m_Game.CurrentTurn == 1 || m_Game.GameType == Game.eGameTypes.PVP)
+            {
+Label: 
+                getRowColFromUser(out row, out col);
+                string msg ="";
```

Comment:
Why the initialization here? you are already initializing "msg" within the "if" statements

Reply:
I init it just in case that no if statement will be true